src/transformers/models/audio_spectrogram_transformer/feature_extraction_audio_spectrogram_transformer.py

```diff
@@ -198,3 +235,16 @@ def __call__(
             padded_inputs = padded_inputs.convert_to_tensors(return_tensors)
 
         return padded_inputs
+
+    def to_dict(self):
```

Comment:
Is this method strictly necessary? If it is, shouldn't it go in the base `FeatureExtractionMixin` class? Rather than copying it out for every feature extractor?

Reply:
which method do you mean?   to_dict?

Reply:
Yes it's about time we add this to the base feature class! 
(it's necessary if we support the numpy part)